inference_rules.adoc

```diff
@@ -610,6 +610,12 @@ The following techniques are disallowed:
 * Techniques that only improve performance when there are identical
   samples in a query. For example, sorting samples in SSD.
 
+== Audit
+After submission, the submitters working group may request that an Available submission be audited. The audit will be funded by MLCommons with a limited time budget (in 0.7, at most 40 hours) and will be completed within 1 month of publication. An auditor will be selected by the review committee who has no conflict of interest with the audited submitter. 
+
```

Comment:
We discussed this at the meeting. Which category or categories (available, preview, and research) are subject to audit?

Reply:
My understanding is that our conclusion was Available submissions only, as captured above in "may request that an Available submission be audited."